setup.py

```diff
@@ -24,10 +24,12 @@
     'psutil',
     'pyprind',
     'python-dateutil',
+    'torch',
```

Comment:
Can we pin the version of pytorch? This is a very important dependency.

Reply:
Will do!